app/controllers/simplified-account/settings/index.controller.js

```diff
@@ -0,0 +1,20 @@
+const formatSimplifiedAccountPathsFor = require('../../../utils/simplified-account/format/format-simplified-account-paths-for')
+const paths = require('../../../paths')
+const { LIVE } = require('../../../models/go-live-stage')
+
+function get (req, res) {
+  const account = req.account
+  const service = req.service
+  // the default setting for the index view is dependent on the account type and go live state
+  if (account.type === 'test' && service.currentGoLiveStage !== LIVE) {
+    return res.redirect(formatSimplifiedAccountPathsFor(paths.simplifiedAccount.settings.serviceName.index, req.account.service_id, req.account.type))
+  } else if (account.type === 'live') {
+    return res.redirect(formatSimplifiedAccountPathsFor(paths.simplifiedAccount.settings.serviceName.index, req.account.service_id, req.account.type))
+  } else {
+    return res.redirect(formatSimplifiedAccountPathsFor(paths.simplifiedAccount.settings.emailNotifications.index, req.account.service_id, req.account.type))
```

Comment:
I don't think we've taken into account the user permission here. We also want to show the email notifications page if the is a non-admin user. 
I think for this entire `get` function the logic could be:

```
if ( (user is non admin) OR (service is live AND account type is 'test') ) {
  redirect to email notifications page
} else {
  redirect to service name page
}
```